react/features/base/config/functions.any.ts

```diff
@@ -89,7 +89,47 @@ export function getFeatureFlag(state: IReduxState, featureFlag: string) {
  * @returns {boolean}
  */
 export function getDisableRemoveRaisedHandOnFocus(state: IReduxState) {
-    return state['features/base/config']?.disableRemoveRaisedHandOnFocus || false;
```

Comment:
The config reducer should allow reading the old value and setting it in the new format.